src/App.tsx

```diff
@@ -1,27 +1,113 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './components/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  // eslint-disable-next-line
-  console.log(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type Props = {};
+
+type State = {
+  isClockVisible: boolean,
+  x: number,
```

Comment:
why do you need `x` in state? What does `x` means btw?

Reply:
renamed it for more clear definition. It`s not a part of the task, variable keeps the amount of time App component has been mounted. 
Do You want me to remove it? It`s not affecting clock component in any way.